gprofiler/main.py

```diff
@@ -791,6 +801,26 @@ def parse_cmd_args() -> configargparse.Namespace:
     if args.nodejs_mode in ("perf", "attach-maps") and args.perf_mode not in ("fp", "smart"):
         parser.error("--nodejs-mode perf or attach-maps requires --perf-mode 'fp' or 'smart'")
 
+    if args.profile_spawned_processes and len(args.pids_to_profile) > 0:
+        parser.error("--pids is not allowed when profiling spawned processes")
+
+    pids_to_profile = []
+    try:
+        if len(args.pids_to_profile) > 0:
+            for input in args.pids_to_profile:
+                pids = [int(pid) for pid in input.split(",")]
+                for pid in pids:
+                    try:
+                        Process(pid)
+                        pids_to_profile.append(pid)
+                    except NoSuchProcess:
+                        continue
+            if len(pids_to_profile) == 0:
+                parser.error("There aren't any alive processes from provided PID list")
```

Comment:
This needn't be a `parser.error` - the arguments are correct but the runtime checks fail.

Logically I want this check to be in `verify_preconditions` and result in a print to stderr, then `sys.exit(1)`.